### DISCUSSION TITLE
Advisory bicycle lane = 2-1 road?

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#4596] [Feedback]
I don't see Denmark in the list on https://github.com/streetcomplete/countrymetadata/blob/master/data/hasAdvisoryCycleLane.yml
But it seems to me that the dotted lanes on a '2-minus-1 road' are basically 'Advisory bicycle lanes'?

https://en.wikipedia.org/wiki/2-1_road

**Comment by westnordost** [comment]
Then it looks like it is missing. You can create a PR over at that repository to add Denmark. Just be sure to add a Danish resource (best: actual legislation text) for verifiability and to see how it is marked.

The research done so far on where such suggestion/advisory cycle lanes exist does not claim to be complete.

**Reply by westnordost** [reply]
I did this now. Added Denmark and Sweden. Let me know if you know if this exists in any other country (Finland, Norway?)

**Reply by ERYpTION** [reply]
Thank you. Not sure about Finland and Norway. 
Although it is now used in many places, the '2-minus-1 road' is a fairly recent thing in Denmark. So I am not sure if people in general know it by that name. But on the other hand I don't think we have a Danish commonly used term for 'Advisory bicycle lane' so I added '2-minus-1 road' as the Danish translation. Hopefully the image used in the quest will look somewhat like a '2-minus-1 road'.
